templates/AWSServiceRoleForEC2Spot.yaml

```diff
@@ -0,0 +1,18 @@
+AWSTemplateFormatVersion: 2010-09-09
+Resources:
+  AWSServiceRoleForEC2Spot:
+    Type: AWS::IAM::Role
+    Properties:
+      RoleName: AWSServiceRoleForEC2Spot
```

Comment:
remove this line.  it's better to let cloudformation manage names.

Reply:
Not a good idea in this case. This needs to be named an exact thing.